lunatrace/bsl/frontend/src/store/slices/authentication.ts

```diff
@@ -85,6 +87,20 @@ export const selectRegisterFlow = (state: RootState) => state.auth.registerFlow;
 export const selectUserId = (state: RootState) => state.auth.session?.identity.id;
 export const selectConfirmedUnauthenticated = (state: RootState) => state.auth.confirmedUnauthenticated;
 
+const sidebarInfoResult = api.endpoints.GetSidebarInfo.select();
```

Comment:
As discussed we are completely getting rid of this function and moving it into either: The sidebar.tsx component itself or a helper